python/ray/autoscaler/commands.py

```diff
@@ -180,18 +188,76 @@ def get_or_create_head_node(config, no_restart, yes):
                                        provider.external_ip(head_node)))
 
 
-def get_head_node_ip(config_file):
+def attach_cluster(config_file, start, override_cluster_name):
```

Comment:
Any reason for choosing `screen` over `tmux`?

Reply:
I think tmux isn't installed on machines by default.